src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -53,7 +53,7 @@ export class CalciteSlider implements LabelableComponent, FormComponent {
   /** Indicates if a histogram is present */
   @Prop({ reflect: true, mutable: true }) hasHistogram = false;
 
-  /** Display a histogram above the slider */
+  /** A list of [x-axis, y-axis] points that illustrates the histogram chart within the slider min and max range */
```

Comment:
This definitely helps better describe the prop, thanks @y0n4. I have a small change request after looking at this again:
- slight verbiage change for clarity
- add `@see` JSDoc tag which we use to add markdown links to the type definition over in Calcite's Developer docs

Does this sound good to you @macandcheese and @benelan?

```tsx
  /**
   * List of x,y coordinates within the slider's min and max, displays above the slider track.
   * @see [DataSeries](https://github.com/Esri/calcite-components/blob/master/src/components/calcite-graph/interfaces.ts#L5)
  */
```

Reply:
SGTM